### ISSUE TITLE
Documentation request: What does colors / sizes mean?

### ISSUE THREAD

**Issue by sesam** [gappleto97/dailyblockchain.github.io#23]
First attempt:

Transactions move slowly out from the center. New transactions that tie in with a visible transaction pulls that transaction in towards the center.
Red is outputs, green is inputs (right?) and lines tie them together. Sizes mean ... ?

Please revise and add, maybe as a tooltip on a "Chart explanation" text somewhere.

**Comment by LivInTheLookingGlass** [comment]
That's correct. This documentation is already in there though, but I agree it's sparse.

It would be fantastic if you could come up with a PR for this. I'm sorry for not responding sooner. I've been spending all my time on [py2p/js2p](https://github.com/gappleto97/p2p-project), and I don't expect that to change anytime soon. Eventually that will be incorporated into this project as I try to get more information into the system.
